README.md

```diff
@@ -526,32 +452,45 @@ fireEvent.click(getElementByText('Submit'), rightClick)
 // default `button` property for click events is set to `0` which is a left click.
 ```
 
-### `prettyDOM`
+### `dom-testing-library` APIs
 
-This helper function can be used to print out readable representation of the DOM
-tree of a node. This can be helpful for instance when debugging tests.
+`react-testing-library` is built on top of
+[`dom-testing-library`](https://github.com/kentcdodds/dom-testing-library) and
+re-exports everything from `dom-testing-library`. Some notable included exports:
 
-It is defined as:
+#### `waitForElement`
 
-```typescript
-function prettyDOM(node: HTMLElement, maxLength?: number): string
+> [Read full docs from `dom-testing-library`](https://github.com/kentcdodds/dom-testing-library/blob/master/README.md#waitforelement)
+
+```js
+import {render, waitForElement} from 'react-testing-library'
+
+test('waiting for an element', async () => {
+  const {getByText} = render(<SearchForm />)
+
+  await waitForElement(() => getByText('Search'))
+})
 ```
 
-It receives the root node to print out, and an optional extra argument to limit
-the size of the resulting string, for cases when it becomes too large.
+#### `wait`
+
+> [Read full docs from `dom-testing-library`](https://github.com/kentcdodds/dom-testing-library/blob/master/README.md#wait)
 
-This function is usually used alongside `console.log` to temporarily print out
-DOM trees during tests for debugging purposes:
+It's recommended to prefer `waitForElement`, but this can be helpful on occasion
 
 ```javascript
-import {render, prettyDOM} from 'react-testing-library'
+import 'jest-dom/extend-expect'
+import {render, wait} from 'react-testing-library'
 
-const HelloWorld = () => <h1>Hello World</h1>
-const {container} = render(<HelloWorld />)
-console.log(prettyDOM(container))
-// <div>
-//   <h1>Hello World</h1>
-// </div>
+test('can fill in the form after loaded', async () => {
+  const {getByLabelText} = render(<Login />)
+
+  // wait until the callback does not throw an error. In this case, that means
+  // it'll wait until we can get a form control with a label that matches "username"
+  await wait(() => expect(queryByText(/loading.../i).not.toBeInTheDOM())
```

Comment:
1) _it'll wait until we can get a form control with a label that matches "username"_ does not sound aligned with the code it comments: it actually waits for "loading..." to disappear from the DOM, and only after that we look for the form control on the next line
2) `/loading.../i` – the `...` matches three arbitrary characters instead of the expected three ASCII dots, worth escaping?